.github/workflows/build.yml

```diff
@@ -23,10 +23,10 @@ jobs:
           coverage: none
 
       - name: Install dependencies
-        run: composer install
+        uses: ramsey/composer-install@v2
 
       - name: Update PHPUnit
         run: composer update phpunit/phpunit --with-dependencies
 
       - name: Run test suite
-        run: vendor/bin/phpunit --exclude-group mayignore --coverage-clover build/logs/clover.xml
```

Comment:
Please keep the code coverage report option.

It will be used when I integrate the `php-coveralls` step during the GitHub action running in the upcoming PR.